### ISSUE TITLE
Suggest to move the action icons to top left

### ISSUE THREAD

**Issue by maxlibin** [Sketch-sh/sketch-sh#311]
This is UX suggestion to move the action icons to the top left where currently feels that these action icons kind of unattached to this container when text is short and there are lots of empty space, it makes different to understand which box or container this actions icons belongs to. For example:
![Screenshot 2022-01-07 at 2 39 44 PM](https://user-images.githubusercontent.com/1277038/148502884-779f6a32-6972-4b3f-9efc-936cb124f164.png)

Small suggestion:
![Screenshot 2022-01-07 at 2 35 37 PM](https://user-images.githubusercontent.com/1277038/148502911-424dcbc2-4dcf-4183-8a47-1cc015de69fa.png)

**Comment by maxlibin** [comment]
I can create a draft of this change if make sense.

**Comment by jchavarri** [comment]
> I can create a draft of this change if make sense.

@maxlibin yes please. As reference, we can also check observable. It has much more functionality than what we need, but they also have the menu on top left corner, e.g. https://observablehq.com/@mbostock/html-in-markdown

![observable](https://user-images.githubusercontent.com/220424/152528818-836ff472-f0ac-4f4c-9a5c-ddf78a83f908.gif)
